Makefile

```
@@ -60,11 +60,11 @@ bin/minio:
 	@echo 'module deps' > bin/go.mod
 	@echo 'require github.com/minio/minio latest' >> bin/go.mod
 	@echo 'replace github.com/minio/minio => github.com/brimsec/minio v0.0.0-20201019191454-3c6f24527f6d' >> bin/go.mod
-	@cd bin && GOBIN=$(CURDIR)/bin go install github.com/minio/minio
+	@cd bin && GOBIN="$(CURDIR)"/bin go install github.com/minio/minio
 
 generate:
-	@GOBIN=$(CURDIR)/bin go install github.com/golang/mock/mockgen
-	@PATH=$(CURDIR)/bin:$(PATH) go generate ./...
+	@GOBIN="$(CURDIR)"/bin go install github.com/golang/mock/mockgen
+	@PATH="$(CURDIR)"/bin:"$(PATH)" go generate ./...
```

For readability, I suggest tweaking all of these additions to use a single pair of quotes around the entire right-hand side.
```suggestion
	@GOBIN="$(CURDIR)/bin" go install github.com/golang/mock/mockgen
	@PATH="$(CURDIR)/bin:$(PATH)" go generate ./...
```

---

@nwt: Thanks for the approval. Indeed, it crossed my mind to go that route, and I was confident that whichever direction I went, the PR feedback would suggest me to go the other direction. I've pushed a change to use quotes around the entire RHS as you suggested. I'll leave it open for a bit just in case someone wants to flip it back the other way again. It's that kinda day. 😉